src/classes/PPlan_Primary_TDTM.cls

```diff
@@ -63,14 +64,13 @@ public with sharing class PPlan_Primary_TDTM extends TDTM_Runnable {
                 }
             }
         }
-        // BEFORE UPDATE
-        if ( newlist != null && triggerAction == TDTM_Runnable.Action.BeforeUpdate && !TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.PPlan_Primary_TDTM_Before_Update)) {
-            for (integer i=0; i<newlist.size(); i++) {
+
+        // AFTER UPDATE
+        if ( newlist != null && triggerAction == TDTM_Runnable.Action.AfterUpdate && !TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.PPlan_Primary_TDTM_After_Update)) {                
```

Comment:
I'm not going to make you change this now as we've been in and out of this file, but in the future, unless there are objections, I really wanna cut down on the calling of methods inside if conditions unless they're super short. 

`!TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.PPlan_Primary_TDTM_After_Update)` is just way to long to be readable inside the condition. I'd prefer setting that value to a boolean first and then passing it in if we can.

Reply:
Sounds like a reasonable suggestion to me, @ryanguest! Like you say, not something I would halt a PR for, specially if it's the only thing, but a good recommendation for the future health of the codebase.

Reply:
I don't have a problem with what you are suggesting...I do, however, want to point out that this was existing code, that I refactored from before to after.  If you want it changed, though, I am happy to do it!

Reply:
Please rereview.  I pushed an update for this...

Reply:
hah, approved but I'll double check! And yea we do that all over the place which is why I didn't wanna hold this up any longer.